source/renderer/app/api/api.js

```diff
@@ -309,17 +309,16 @@ export default class AdaApi {
         daedalusTotal: total,
         requestDurationInMs: moment.duration(moment().diff(requestTimestamp)).as('milliseconds'),
       };
-      // eslint-disable-next-line max-len
-      Logger.debug(`AdaApi::searchHistory success: ${total} transactions loaded ${stringifyData(responseStats)}`);
+      Logger.debug(`AdaApi::searchHistory success: ${total} transactions loaded`, { ...responseStats });
       return new Promise(resolve => resolve({ transactions, total }));
     } catch (error) {
-      Logger.error(`AdaApi::searchHistory error: ${stringifyData(error)}`);
+      Logger.error('AdaApi::searchHistory error', { error: `${stringifyError(error)}` });
       throw new GenericApiError();
     }
   };
 
   createWallet = async (request: CreateWalletRequest): Promise<Wallet> => {
-    Logger.debug('AdaApi::createWallet called');
+    Logger.debug('AdaApi::createWallet called', { parameters: request });
```

Comment:
@MarcusHurney here we need to be very careful not to log parameters which may contain sensitive information such as spending password or mnemonics. This is why in the original error message we didn't log the request.